### ISSUE TITLE
Tells me to login to Azure account, opens tab that says its successful, then says it's unable to connect

### ISSUE THREAD

**Issue by airoeva** [microsoft/vscode-tools-for-ai#1619]
Does this occur consistently? 
Repro steps:

1.
2.

Action: azureAccount.onSessionsChanged
Error type: 123
Error Message: Unknown Error retrieving susbcriptions from Azure Account extension

Version: 0.14.0
OS: linux
OS Release: 5.4.0-1077-azure
Product: Visual Studio Code
Product Version: 1.69.2
Language: en

<details>
<summary>Call Stack</summary>

```
b.<anonymous> extension.js:2:2060849
s extension.js:2:2057717
```

</details>

**Comment by shsuman** [comment]
@tbombach Can you please look into it ?

**Comment by jedcheng** [comment]
Hi there

This problem can be got around by signing in through the Azure plugin, instead of the Azure ML plugin. 

Are you still having this issue?

I got in touch with a Microsoft engineer about the exact same problem last month (on a Mac with Chrome).

After that, it seems that there is a recent update with VS code or Electron (I am not sure) that fixed this problem for Chrome. 

We then tested it for Safari and Chrome to sign in. Using Safari doesn't help but using Chrome does work.
